python/src/aiconfig/editor/client/src/components/prompt/model_settings/ModelSettingsRenderer.tsx

```diff
@@ -11,11 +11,21 @@ type Props = {
   onUpdateModelSettings: (settings: Record<string, unknown>) => void;
 };
 
+const useStyles = createStyles(() => ({
+  settingsContainer: {
+    maxHeight: "400px",
```

Comment:
nit: should we use something else other than px? Ex: %

Reply:
Potentially, but just doing this for now and we can update later with our UI/style pass through